script/build.fsx

```diff
@@ -20,13 +21,13 @@ let releaseNotes =
     |> ReleaseNotesHelper.parseReleaseNotes
 
 Target "Clean" (fun _ ->
-    CleanDirs [buildDir; testResultsDir]
+    CleanDirs [buildDir; packagingDir; testResultsDir]
 )
 
 open Fake.AssemblyInfoFile
 
 Target "AssemblyInfo" (fun _ ->
-    CreateCSharpAssemblyInfo ".\SolutionInfo.cs"
+    CreateCSharpAssemblyInfo "./SolutionInfo.cs"
```

Comment:
Looked like most other paths in the script were using `/` over `\` so I just changed this for that reason.